persistence/record/src/main/scala/net/liftweb/record/MetaRecord.scala

```diff
@@ -148,7 +148,7 @@ trait MetaRecord[BaseRecord <: Record[BaseRecord]] {
 
     fieldList = {
       val ordered = fieldOrder.flatMap(f => tArray.find(_.metaField == f))
-      ordered ++ (tArray -- ordered)
+      ordered ++ (tArray --= ordered)
```

Comment:
This is a weird one that we should look at in a follow-up PR (and before we cut a non-SNAPSHOT release from this), I think.